src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
+    private String fruit;
+    private int quantity;
+    private String type;
+
+    public FruitTransaction(String fruit, int quantity) {
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
```

Comment:
This constructor is not setting the `type` field. Every transaction should have a type associated with it, so it might be better to remove this constructor to avoid incomplete object states.